operations/configuration_management/CFS_Key_Management.md

```diff
@@ -0,0 +1,33 @@
+## CFS Key Management and Permission Denied Errors
+
+Configuration Framework Service \(CFS\) manages it's own keys separate from keys for communication between CFS and the components or images that it is configuring.  These are separate from the keys used by users and should not need to be managed.
+
+### Permission Denied Errors
+
+If Ansible is unable to connect with it's target and fails with an `Unreachable - Permission denied` error the first place to check is the `cfs-state-reporter` on the target node.
+
+Every booted node should be running a copy of cfs-state-reporter.  This service is responsible for pulling down the public key.  To check the status of this service, ssh to the node that CFS can't communicate with, and run `systemctl status cfs-state-reporter`. 
+
+The cfs-state-reporter should be complete, but report success.  Any other state
+can be an indication of a problem.
+
+```
+ncn-m001:~ # systemctl status cfs-state-reporter
```

Comment:
Command prompt should be `ncn-m001#`

Reply:
Is this just our standard?  These were copied from the terminal, so they're accurate output.

Reply:
It is a documentation standard. There is more information on this page if you are interested: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#markdown-format

Reply:
fixed